Patches/Dragonian Race/ThingDefs_Misc/DR_Apparel.xml

```diff
@@ -65,6 +65,60 @@
 				</value>
 			</li>
 			
+			<!-- ========== Armor ========== -->
+			
+			<li Class="PatchOperationReplace">
```

Comment:
You've confirmed that the combination of material armor + statBases armor produces the range of armor values you want?

Reply:
Yes.